examples/getting_started/web_search.ipynb

```diff
@@ -0,0 +1,255 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Web Search Agent Implementation\n",
+        "\n",
+        "This notebook demonstrates the implementation of a Web Search Agent using the Mirascope library. The agent is capable of performing web searches and parsing webpage content to answer user queries.\n",
+        "\n",
+        "## Table of Contents\n",
```

Comment:
we can probably remove "Table of Contents" here and just have the table of contents right under the header?